docs/en-US/component/input-number.md

```diff
@@ -103,6 +111,13 @@ input-number/controlled
 | value-on-clear ^(2.2.0) | value should be set when input box is cleared    | ^[number] / ^[null] / ^[enum]`'min' \| 'max'` | —         |
 | validate-event          | whether to trigger form validation               | ^[boolean]                                    | true      |
 
+### Slots
+
+| Name    | Description                  |
+| ------- | ---------------------------- |
+| prepend | custom input box front icon  |
```

Comment:
```suggestion
| prepend ^(2.6.3) | custom input box front icon  |
```

Reply:
Do you think `decrease` is better then `prepend`?

Reply:
> Do you think is better then ?`decrease``prepend`

It's not impossible either, it's just that following the naming conventions of similar components would be more convenient to remember

Reply:
I know you refered to el-input. But the function is not the same. The icon slot in el-input-number only used for replacing the decrease and increase icon, It's not universal. @btea What do you think?
![image](https://github.com/element-plus/element-plus/assets/38392315/d0427059-0b63-4547-b009-8ee60ddfc585)

Reply:
It makes sense. In fact, there are cases where these two icons are on the right side at the same time. In this case, the name suggested by @kooriookami  seems clearer.

Reply:
Maybe it can be clearer, such as `increase-icon`/`decrease-icon`.

Reply:
okay